keyboards/monsgeek/m2/config.h

```diff
@@ -0,0 +1,35 @@
+/* Copyright (C) 2023 jonylee@hfd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* LED Indicators */
+#define LED_WIN_LOCK_PIN        C11
+
+/* SPI Config for spi flash*/
+#define SPI_DRIVER SPIDQ
+#define SPI_SCK_PIN B3
+#define SPI_MOSI_PIN B5
+#define SPI_MISO_PIN B4
+#define SPI_MOSI_PAL_MODE 5
+
+#define EXTERNAL_FLASH_SPI_SLAVE_SELECT_PIN C12
+
+/* I2C Config for LED Driver */
+#define SNLED27351_I2C_ADDRESS_1 SNLED27351_I2C_ADDRESS_GND
+#define SNLED27351_I2C_ADDRESS_2 SNLED27351_I2C_ADDRESS_VDDIO
+#define I2C1_OPMODE OPMODE_I2C
```

Comment:
This is the default value, and doesn't need to be set here.
```suggestion
```

Reply:
Hi d.  This is true for STM mcu's .. it's set in [i2c_master.c](https://github.com/qmk/qmk_firmware/blob/master/platforms/chibios/drivers/i2c_master.c#L43-L46) but I don't think `USE_I2CV1` is set on the WB32 - at least it's not on my clone.